H2TauTau/python/heppy/analyzers/TauP4Scaler.py

```diff
@@ -0,0 +1,40 @@
+from PhysicsTools.Heppy.analyzers.core.Analyzer import Analyzer
+from PhysicsTools.Heppy.analyzers.core.AutoHandle import AutoHandle
+from CMGTools.H2TauTau.heppy.utils.TauEnergyScales import TauEnergyScales
+
+class TauP4Scaler(Analyzer):
+
+    gen_match_dict = {6:'JetToTau',
```

Comment:
est-ce que ceci est defini seulement une fois? 
il me semblait qu'on avait defini ca ailleurs. 
Si la definition de la provenance des taus doit etre utilisee a plusieurs endroits, la definir de maniere unique, probablement la ou tau.gen_match est obtenu. 

meme commentaire pour les decay modes

Reply:
Cela fait partie de devellopements prevu qui viendront dans une prochaine PR, c.f. this issue : https://github.com/GaelTouquet/cmgtools-lite/issues/3